### ISSUE TITLE
Player & Editor Menu Bar?

### ISSUE THREAD

**Issue by BKFighter** [Newawe/player#2]
Should we add the main menu bar thing to the Player and Editor also? (Or do we want something else similar on them?)

**Comment by skistaddy** [comment]
Yeah, we should. (Note: I'm still going to be around, just to give my opinion on stuff.)

**Comment by haylinmoore** [comment]
@BKFighter Once we combine them then we will have the main menu bar. We are waiting for @robinp7720 to rework the page systems

**Comment by BKFighter** [comment]
@bjskistad Hi!

@herohamp Ok.

**Comment by skistaddy** [comment]
@BKFighter (Hi. :) P.S. Your my favorite person ever besides my parents and @herohamp and a few select others in life including my friends in real life.)

**Comment by BKFighter** [comment]
@bjskistad That's such a nice thing of you to say! Thanks! Anyways @herohamp I have time to now.

**Comment by haylinmoore** [comment]
Thank @bjskistad

**Comment by skistaddy** [comment]
@BKFighter Someone edited that. I'm not sure who. :/ (See the edited thing at the top.) (Don't worry, you're still one of my favorites. :D)

**Comment by skistaddy** [comment]
@BKFighter Very funny. I saw the audit log. You did it. XD

**Comment by BKFighter** [comment]
hehe LOL I've been waiting for a good opportunity.

**Comment by skistaddy** [comment]
Wow. XD

**Comment by jeandrek** [comment]
What is the audit log?

**Comment by BKFighter** [comment]
@Jonathan50 I'm assuming he meant edit log. (Unless we're being audited... Scary thoughts)

**Comment by skistaddy** [comment]
@Jonathan50 The audit log is a giant list full of everything that happens in all of the Newawe repos. 0_0
